nautobot/core/views/generic.py

```diff
@@ -197,10 +200,39 @@ def get(self, request):
         content_type = ContentType.objects.get_for_model(model)
 
         filter_params = request.GET
+        user = request.user
         display_filter_params = []
         dynamic_filter_form = None
         filter_form = None
         hide_hierarchy_ui = False
+        clear_view = filter_params.get("clear_view", False)
+
+        # If the user clicks on the clear view button, we do not check for global or user defaults
+        if not clear_view:
+            # Check if there is a default for this view for this specific user
+            app_label, model_name = model._meta.label.split(".")
+            view_name = f"{app_label}:{model_name.lower()}_list"
+
+            if not isinstance(user, AnonymousUser):
+                user_default_saved_view_pk = user.config_data.get("saved_views", {}).get(view_name, None)
+                if user_default_saved_view_pk is not None and not request.GET.get("saved_view"):
+                    try:
+                        SavedView.objects.get(pk=user_default_saved_view_pk)
```

Comment:
Should we enforce that either `user=user` or `is_shared=True` on this `get()`? So that a user can't have a default saved view that belongs to another user and isn't shared?

Same question on the equivalent UIViewSet code.

Reply:
Good catch I think that is reasonable.